kuka_kr150_support/package.xml

```diff
@@ -0,0 +1,53 @@
+<?xml version="1.0"?>
+<package format="2">
+  <name>kuka_kr150_support</name>
+  <version>0.1.0</version>
+  <description>
+    <p>
+      ROS-Industrial support for the KUKA KR150-2 (and variants).
+    </p>
+    <p>
+      This package contains configuration data, 3D models and launch files
+      for KUKA KR 150-2 manipulators. This currently includes the KR 150-2 only.
+    </p>
+    <p><b>Specifications</b>:</p>
+    <ul>
+      <li>KR 150-2 - Default</li>
+    </ul>
+    <p>
+      Joint limits and maximum joint velocities are based on the information
+      found in the online datasheet <a href="https://www.kuka.com/-/media/kuka-downloads/imported/6b77eecacfe542d3b736af377562ecaa/pf0020_kr1502_en.pdf">here</a>. All urdfs are based on the default motion and joint velocity limits,
```

Comment:
This link `404`s for me at the moment.

Does it work for you? Perhaps we should refer to the document or datasheet nr. That way it can be searched for.